### ISSUE TITLE
path flag of generator not working

### ISSUE THREAD

**Issue by simon-winter** [go-jet/jet#426]
**Describe the bug**
whatever i write into the path flag for the jet generator has no influence on the destination path. When using the example path ./gen, the resulting files still just get generated into my workspace root with this structure: "DBName/schemaName/..."
there is no .gen folder created. When i create that first or try a different naming, still it is not copyied inside there nor a folder is created. it just cares about the database name and schema name for the path of generation.

**Environment (please complete the following information):**
 - OS: Windows 11
 - Database: postgres

**Comment by jamius19** [comment]
What's the full command you ran?

**Comment by simon-winter** [comment]
Correction: it seems on windows/vscode terminal the "./" infront of the .gen folder is the problem. when using . path the entire path is not used.

 jet -dsn=postgresql://postgres:postgres@localhost:5432/webDev?sslmode=disable -schema=public -path=./test //broken
  jet -dsn=postgresql://postgres:postgres@localhost:5432/webDev?sslmode=disable -schema=public -path=test //working

**Comment by jamius19** [comment]
That's interesting, the relative path with `./` should work. I'll check on a windows machine.

**Comment by simon-winter** [comment]
Yes, tested with powershell, also omited relative path. The debug output of the generator also completly omits the relative path:
'Destination directory: webDev/public' -> missing the "test" folder when provided as relative path.
having non relative path is interpreted as relative path though, meaning the ./ is unnecessary

**Comment by jamius19** [comment]
>The debug output of the generator also completly omits the relative path

That's when the path is `./test ` right?

>having non relative path is interpreted as relative path though, meaning the ./ is unnecessary

I agree, but then again path handling in Windows is a bit different.

Can you please try with this command and let me know?
`jet -dsn=postgresql://postgres:postgres@localhost:5432/webDev?sslmode=disable -schema=public -path=.\test`

**Comment by simon-winter** [comment]
> That's when the path is ./test right?
yes, if the path is relative it seems to get completly ignored and omited.

> Can you please try with this command and let me know?
> jet -dsn=postgresql://postgres:postgres@localhost:5432/webDev?sslmode=disable -schema=public -path=.\test

same behavior, path omited

**Comment by jamius19** [comment]
Interesting, `.\test` on Windows should have worked per my understanding.
I'll look into it.

**Comment by jamius19** [comment]
Confirmed relative names starting with `./` and `.\` don't work.

**Comment by jamius19** [comment]
This is related to how PowerShell parses arguments.
The value of the `path` flag is empty in Jet when it's passed like this,
```shell
jet.exe -source=sqlite -dsn="main.db" -path=.\dest\dir
```
See [this SO question](https://stackoverflow.com/questions/56775646/pass-a-parameter-that-contains-dots-in-powershell-script) for more context.

### How to pass arguments:
When using PowerShell, the arguments should be quoted like so,
```shell
jet.exe -source="sqlite" -dsn="main.db" -path=".\dest\dir"
```

It'll work as expected then.
\
`cmd` doesn't have any problems either way, but in Windows, I'd err on the side of caution and use quoted strings as arguments anyway.
